src/b2_user_settings.h

```diff
@@ -30,12 +29,14 @@ struct B2_API b2BodyUserData {
 
 struct B2_API b2FixtureUserData {
 	b2FixtureUserData() {
-		shape_idx = 0;
+		shape_idx = -1;
 		box2d_fixture_idx = 0;
+		type = 0;
 	}
 
 	int shape_idx;
 	int box2d_fixture_idx;
+	int type;
```

Comment:
This seems to be unused; I would think it's unnecessary to add here.

Reply:
fixed

Reply:
Looks like it came back. After that fix (and squash) this looks good to merge!